addons/web_tour/static/src/tour_service/tour_utils.js

```diff
@@ -416,6 +427,41 @@ export class RunningTourActionHelper {
         return this.anchor;
     }
 
+    /**
+     * **blur** action by default for input cause troubles mainly when input is in dialog.
+     * That's why we prefer **tab** action for inputs.
+     * @param {Node} node
+     * @returns {"blur"|"enter"|"tab"|null}
+     */
+    _get_edit_confirm_action = (node) => {
+        const tagNode = getTag(node, true);
+        const confirmActions = {
+            input: [
+                ["class", "o_searchview_input", null],
+                ["parentClass", ".o_list_view", "click .o_selected_row"],
+                ["parentClass", "we-list", "blur"],
+                ["id", "o_link_dialog_url_input", null], // website
```

Comment:
couldn't this one be dealt with user space (so a specific run in the tour itself)?